documentation/convert_notebooks.py

```diff
@@ -0,0 +1,130 @@
+import os
+from datetime import datetime
+import argparse
+import requests
+import yaml
+from nbconvert import MarkdownExporter
+from yaml.loader import BaseLoader
+
+with open('authors.yml', 'r') as f:
+    mapping = yaml.load(f, Loader=BaseLoader)
+
+def convert_notebook_to_md(input_file):
+    """
+    convert notebook (.ipynb) file to markdown format
+
+    :param input_file: jupyter notebook
+    :type input_file: str
+    :return: converted md file
+    :rtype: str
+    """
+    md_exporter = MarkdownExporter()
+    nb_body, _ = md_exporter.from_filename(input_file)
+    return nb_body
+
+
+def get_github_author(github_path, owner, repo): 
+    """
+    given a file on github, find all authors' github id in chronological order, no duplicate
+    (github authors are required metadata for Azure Doc)
```

Comment:
I think we already have this file, consider just modifying it to add your deployment target through an argument or something like that